llarp/config/config.cpp

```diff
@@ -61,8 +61,8 @@ namespace llarp
         "netid",
         Default{llarp::LOKINET_DEFAULT_NETID},
         Comment{
-            "Network ID; this is '"s + llarp::LOKINET_DEFAULT_NETID
-                + "' for mainnet, 'gamma' for testnet.",
+            "Network ID; this is '"s + llarp::LOKINET_DEFAULT_NETID + "' for mainnet, "s
+                + llarp::LOKINET_TESTNET_NETID + "for testnet."s,
```

Comment:
Needs some reformatting; this is currently becoming:
```ini
# Network ID; this is 'lokinet' for mainnet, testnetfor testnet.
```
in a generated config (`./daemon/lokinet -r -g foo/lokinet.ini`)